metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -599,7 +599,13 @@ class TableauSourceReport(StaleEntityRemovalSourceReport):
     num_datasource_field_skipped_no_name: int = 0
     num_csql_field_skipped_no_name: int = 0
     num_table_field_skipped_no_name: int = 0
+    # lineage
+    num_upstream_table_lineage: int = 0
```

Comment:
is this the number of tableau entities with upstream lineage, or the number of upstream lineage entries across all upstreamLineage aspects?

imo the former would be more useful

Reply:
the second, every time an UpstreamLineage aspect is emitted, the counter is increased